src/common/serial_interface.cc

```diff
@@ -59,7 +59,7 @@ SerialInterface::setup(const char *path)
 }
 
 void
-SerialInterface::setAttributes(int speed)
+SerialInterface::setAttributes(int speed) const
```

Comment:
Can we just turn this const thing off entirely? So much of this codebase deals with wrapping C APIs with const-correct C++ wrappers that the number of special cases outweighs the advantages

Reply:
Yeah ok.